README.md

```diff
@@ -17,6 +17,21 @@ for Ada:
 
 These are currently under development.
 
+Installing
+----------
+If you are only concerned in getting the suite of executables you can most easily do so by using the [Alire][alire] package manager to handle the building and installing for you. The steps are:
+```bash
+alr install libadalang_tools
+```
+The build step will require a substantial amount of CPU and RAM, but once successful you should find the executables in `$HOME/.alire/bin`. If this is not in the `$PATH` set up by your shells resource files, you should do so now, typically doing something like this (example is using Bash):
```

Comment:
```suggestion
alr install libadalang_tools
```
The build step will require a substantial amount of CPU and RAM, but once successful you should find the executables in `$HOME/.alire/bin`. If this is not in the `$PATH` set up by your shell's resource files, you should do so now, typically doing something like this (example is using Bash):
```